dotCMS/src/integration-test/java/com/dotcms/publishing/PublisherAPIImplTest.java

```diff
@@ -1163,24 +1165,69 @@ public void AddAllCategoryIntoBundle() throws Exception {
      * Test delete file works after the security fixes
      */
     @Test
-    public void testDeleteFileDesc() {
+    public void testFilterDescriptors() throws IOException {
```

Comment:
Update javadoc with proper elements: method to test, given scenario, expected result.

Reply:
@dsilvam done!